docs/optimization_guide/nncf/ptq/basic_quantization_flow.md

```diff
@@ -93,14 +92,14 @@ Once the dataset is ready and the model object is instantiated, you can apply 8-
 
 .. note:: The ``model`` is an instance of the ``torch.nn.Module`` class for PyTorch, ``onnx.ModelProto`` for ONNX, and ``openvino.runtime.Model`` for OpenVINO.
 
-After that the model can be exported into th OpenVINO Intermediate Representation if needed and run faster with OpenVINO.
+After that the model can be exported into the OpenVINO Intermediate Representation if needed and run faster with OpenVINO.
```

Comment:
I think we need to have section that shows how to do it. It is not very intuitive.
Moreover, there is description of what model is, but no description of what optimized_model is. Is it OV model? Can I compile it right away?

Reply:
I've added code example. Please check.